app/routes/index.tsx

```diff
@@ -106,6 +106,15 @@ export default function App() {
     .map(({title}) => title)
 
   const handleSpecialLinks = (e: MouseEvent) => {
+    // The AutoHeight component doesn't notice when a HTML details is opened.
+    // Manually removing the height from the style fixes this, but can potentially
+    // break something else...
+    const target = e.nativeEvent.target as HTMLElement
```

Comment:
This I'm very unsure about... But it works, at least for now...

Reply:
if it works for now, let's try it and figure out problems as they come, fortunately it's not AGI Alignment